pt-BR/mail/chrome/messenger/preferences/advanced.dtd

```diff
@@ -19,27 +19,27 @@
 <!ENTITY markAsReadDelay.label         "Ao exibir por">
 <!ENTITY markAsReadDelay.accesskey     "o">
 <!ENTITY secondsLabel.label            "segundos">
-<!ENTITY showFolderColumns.label       "Show expanded columns in the folder pane">
+<!ENTITY showFolderColumns.label       "Exibir colunas expandidas no painel de pastas">
 <!ENTITY showFolderColumns.accesskey   "l">
-<!ENTITY rememberLastMsg.label         "Remember the last selected message">
+<!ENTITY rememberLastMsg.label         "Lembrar a última mensagem selecionada">
 <!ENTITY rememberLastMsg.accesskey     "b">
-<!ENTITY openMsgIn1.label              "New messages should be opened in:">
-<!ENTITY openMsgInNewTab.label          "Nova aba">
+<!ENTITY openMsgIn1.label              "Novas mensagens devem abertas em:">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY openMsgIn1.label              "Novas mensagens devem ser abertas em:">